engine/include/cubos/engine/cubos.hpp

```diff
@@ -39,6 +39,13 @@ namespace cubos::engine
         /// @param tag The tag to be executed after.
         TagBuilder& afterTag(const std::string& tag);
 
+        /// Adds a condition to the current tag. If this condition returns
+        /// false, systems with this tag will not be executed. For the tagged
+        /// systems to run, all conditions must return true.
+        /// @param func The condition function.
+        template <typename F>
+        TagBuilder& addCondition(F func);
```

Comment:
Rename to `runIf`?

Reply:
Forgot about this decision in the meeting, thanks :+1: